sklearn/feature_extraction/tests/test_feature_hasher.py

```diff
@@ -4,6 +4,7 @@
 import pytest
 
 from sklearn.feature_extraction import FeatureHasher
+from sklearn.feature_extraction._hashing import transform as _hashing_transform
```

Comment:
This actually broke PyPy CI, because in PyPy the `_hashing` submodule is not built/supported and so we can't import it at top level. Should be fixed in https://github.com/scikit-learn/scikit-learn/commit/4ef679b082edfb5642764c2a5ed3b6575b0ea5e4 on master.